examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3107,9 +3107,26 @@ server cluster RelativeHumidityMeasurement = 1029 {
 }
 
 server cluster OccupancySensing = 1030 {
-  readonly attribute bitmap8 occupancy = 0;
-  readonly attribute enum8 occupancySensorType = 1;
-  readonly attribute bitmap8 occupancySensorTypeBitmap = 2;
+  enum OccupancySensorType : ENUM8 {
+    kPir = 0;
```

Comment:
this should be PIR instead of Pir (based onspec naming)

@bzbarsky-apple is PIR supposed to be handled as an acronym that is always uppercase?

Reply:
@andy31415 In the Darwin code and APIs, we now preserve acronyms (pass preserveAcronyms=true to asUpperCamelCase) but in general asUpperCamelCase does not do that and will happily convert "PIR" to "Pir", as I recall...